server/openai/generateItenerary.js

```diff
@@ -0,0 +1,56 @@
+const openaiClient = require('./openaiClient')
+
+async function generateItinerary(constraints) {
+  try {
+    const conversation = [
+      {
+        role: 'system',
+        content: `You are an AI that generates travel itineraries. Respond with ONLY JSON that contains the plan for each stage. 
+        ONLY respond with the following format, do not include any descriptions or codeblocks, I should be able to parse the output to a JavaScript Object
+        The response needs to be formatted exactly like the following structure. Add some Food and snack stages as well with an optional: true marker.:
+        '''
+        {
+          plan:[
+            {
+              day: Number
```

Comment:
oh also each stage needs a tailwind color of the format `bg-XXX-400`

Reply:
could we not generate that locally and store? I can ask GPT to send it but im worried it may send invalid values.

Reply:
I see, we can do that locally then.

note to self to add a gpt3.5 data postprocessor that 
1. randomizes a number btwn 0 and the list of valid tailwind colors
2. locally writes item.color to have the tailwind color.js array item corresponding to this number